cmd/atlas/atlas.go

```diff
@@ -37,15 +40,33 @@ var (
 func Execute() {
 	ctx := telemetry.NewContext()
 	rootCmd := atlas.Builder(&profile)
+	initSignalHandler(rootCmd)
```

Comment:
[q] is this tracking always root command or the command fired?

Reply:
Signal handling (trapping) works for the entire application, ie. across all commands. 

Whenever a signal is trapped, an error is created and passed with the rootCmd through to the handleError function , which ensures that the error is tracked in the same way that it is currently being tracked. 

You can try it out for yourself: run "atlas projects list", which will sleep for 1 minute so that you have enough time to press CTRL-C (or CTRL-Z, or CTRL-\) and see that an error is reported in stdout and also tracked.